content/en/videos/video11.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Notary v2 Outstanding Issues Working Session - Justin Cormack"
```

Comment:
Let's remove the speaker's name from the title as the video cover includes the speaker's name. If we keep the speaker name, please make sure all speaker's names are there.

Reply:
Since the video thumbnail includes the speaker's name, there is no need to mention it again. I will look into it.